src/pages/gnana/index.tsx

```diff
@@ -3,10 +3,29 @@ import { GetServerSideProps } from 'next'
 import { getServerSideGenericProps } from 'components/getServersideProps'
 import ApeZone from 'views/ApeZone'
 import { wrapper } from 'state'
+import fetchPrices from 'state/tokenPrices/fetchPrices'
+import { setPoolsPublicData } from 'state/pools'
+import fetchPools from 'state/pools/fetchPools'
 
-export const getServerSideProps: GetServerSideProps = wrapper.getServerSideProps(
-  (store) => async (context) => getServerSideGenericProps({ ...context, store }),
-)
+export const getServerSideProps: GetServerSideProps = wrapper.getServerSideProps((store) => async (context) => {
+  const initialProps = await getServerSideGenericProps(context)
```

Comment:
can you pass store to this function, look at how it's done in [homepage](https://github.com/innovation-upstream/apeswap-frontend/blob/b748bb82305ed6ef088c740a8159b5216415340b/src/pages/index.tsx#L22)